src/test/java/cosminsechel/tests/tema8/TestPage/BaseTest.java

```diff
@@ -0,0 +1,3 @@
+package cosminsechel.tests.tema8.TestPage;
+
```

Comment:
Add implementation for BaseTest. Here you should have before and after methods for setting up drivers for mobile/desktop, closing browsers.